woocommerce/woocommerce-gutenberg-products-block
# Add Global Styles support for the Add To Cart Button block #5666

This PR adds support for global style for the `Add To Cart Button` block.

Now, the user can edit the style for:
- text-color
- background-color
- border-radius
- font-size



#4965 

### Screenshots

![image](https://user-images.githubusercontent.com/4463174/153617092-faebfb9f-6941-46ca-b871-e569de11a2b6.png)
*without global style*

![image](https://user-images.githubusercontent.com/4463174/153616344-f278a557-49d7-4968-838f-79e639849b9b.png)
*with global style*

### Testing

### Manual Testing

1. Be sure that you are on `WordPress 5.9`.
2. Install and enable the `Twenty Twenty-Two` theme.
3. Add the `All Product` block (this block contains `Add To Cart Button Block`) to a post.
4. Get the focus on `Add To Cart Button`.
5. On the right sidebar, personalize the styles of the block.
6. Go on the page and check if there are changes.
7. Reset to default using the `Reset` button from the different sections. 
8. Go to Dashboard and select Appearance > Editor (beta). At the top of the screen, select Home > Browser all templates > Single Post. When the page is loaded, add the block to the page.
9. On the Editor page click on the `Styles` icon on the right-top corner.
10. Verify that the `Add To Cart Button` is shown under the `Blocks` section. Personalize again the block.
11. Save your changes.
12. Go on the page created earlier and check if all styles are applied correctly.
13. Edit your previous post/page again.
14. Change again the styles.
15. Save your changes.
16. Check if these styles have priority over the styles from the Site Editor.


### Changelog

> Add Global Styles support to the Add To Cart Button block.
